test/README.md

```diff
@@ -1,72 +1,38 @@
-# Testing the WebSDK
+# Running WebSDK Test Suite
 
-## Testing Commands
-1. Run all and watch file changes: `yarn test`
-2. Run all with no watch: `yarn test:noWatch`
-3. Run specific file: `<one of above commands> <path to transpiled file>` e.g: `yarn test:noWatch build/ts-to-es6/test/unit/managers/ConfigManager.js`
+## VSCode Run a specific test
+_Tested up to VSCode Version: `1.54.2`_
+1. Install the ["Remote Development"](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) VS Code Extension
+   - Restart VSCode
+2. You will be prompted that there is a Dev Container detected, press "reopen in a container"
+   - If no prompt, press F1 and enter "Remote-Containers: Open Folder in Container.."
+3. Install [Testify](https://marketplace.visualstudio.com/items?itemName=felixjb.testify) VS Code Extension
+4. Go to the test file you want to run and click "Run Test" above the name.
```

Comment:
I'm not seeing this above the file name. Perhaps provide a screenshot to be clearer.